content/filters/array_filters.md

```diff
@@ -0,0 +1,186 @@
+# Array Filters
+
+Array filters are used to modify the output of arrays.
+
+1. TOC
+{:toc}
+
+## join
+
+Joins the elements of an array with the character passed as the parameter. The result is a single string.
+
+#### Input
+
+~~~ liquid
+  {{ product.tags | join: ', ' }}
+~~~
+
+#### Output
+
+~~~
+tag1, tag2, tag3
+~~~
+
+## first
+
+Returns the first element of an array.
+
+#### Input
+
+~~~ liquid
+  <!-- product.tags = "sale", "mens", "womens", "awesome" -->
+    {{ product.tags | first }}
+~~~
+
+#### Output
+
+~~~
+  sale
+~~~
+
+<br>
+
+`first` can be used in dot notation, in cases where it needs to be used inside a [tag](/tags/).
+
+~~~ liquid
+  {% if product.tags.first == "sale" %}
+      This product is on sale!
+  {% endif %}
+~~~
+
+## last
+
+Gets the last element passed in an array.
+
+#### Input
+
+~~~ liquid
+  <!-- product.tags = "sale", "mens", "womens", "awesome" -->
+    {{ product.tags | last }}
+~~~
+
+#### Output
+
+~~~
+awesome
+~~~
+
+<br>
+
+`last` can be used in dot notation, in cases where it needs to be used inside a [tag](/tags/).
+
+~~~ liquid
+  {% if product.tags.last == "sale"%}
+      This product is on sale!
+  {% endif %}
+~~~
+
+<br>
+
+Using `last` on a string returns the last character in the string.
+
+#### Input
+
+~~~ liquid
+  <!-- product.title = "Awesome Shoes" -->
+  {{ product.title | last }}
+~~~
+
+#### Output
+
+~~~
+  s
+~~~
+
+## index
+
+Returns the item at the specified index location in an array. Note that array numbering starts from zero, so the first item in an array is referenced with `[0]`.
+
+#### Input
+
+~~~ liquid
+  <!-- product.tags = "sale", "mens", "womens", "awesome" -->
+    {{ product.tags[2] }}
```

Comment:
No idea why pygments is colorizing it.
You could try applying a different lexer from here http://pygments.org/languages/
Python would be good, but its coloring the html comment as well...

Or we could switch the `err` class coloring, we dont want to shwo errors anyway on production :wink: